PR_DESCRIPTION.md

```
@@ -0,0 +1,193 @@
+    grep -q "$pattern" "$file"
+    exit_code=$?
+    set -e
+    check $exit_code "$message"
+}
+```
+
+## ✅ 测试结果
+
+### 自动化测试
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)